### PR TITLE
Transactional templates support

### PR DESCRIPTION
**Pull request by gondzo**
- Transactional templates support 
- Remove placeholders service

NOTE: all templates should be migrated to transactional (v3) templates before deployment

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -1,12 +1,6 @@
 'use strict';
 
 module.exports = {
-  '/templates/:name': {
```
Comment (sachin-maheshwari) [inline src/routes.js]: @gondzo please clean "docs" folder as well.

**Comment by gondzo** [comment]
sure. We have no api endpoints except for health. should we just remove swagger and postman files? or just add the health endpoint?

**Comment by sachin-maheshwari** [comment]
@gondzo, yes delete other endpoints and add just add health check endpoint . Also please correct retry logic as discussed earlier.

**Comment by gondzo** [comment]
@sachin-maheshwari PR updated
Added EMAIL_RETRY_MAX_AGE variable that defaults to 24h

**Pull request merged**